docs/getting-started/linux-manual.md

```diff
@@ -35,27 +35,57 @@ installed on x86_64/amd64 platforms.
 
 1. Set up the package repository and install the collector package:
 - Debian:
-```sh
-curl -sSL https://splunk.jfrog.io/splunk/otel-collector-deb/splunk-B3CD4420.gpg > /etc/apt/trusted.gpg.d/splunk.gpg
-echo 'deb https://splunk.jfrog.io/splunk/otel-collector-deb release main' > /etc/apt/sources.list.d/splunk-otel-collector.list
-apt-get update
-apt-get install -y splunk-otel-collector
-```
-- RPM:
-```sh
-yum install -y libcap  # Required for enabling cap_dac_read_search and cap_sys_ptrace capabilities on the collector
-
-cat <<EOH > /etc/yum.repos.d/splunk-otel-collector.repo
-[splunk-otel-collector]
-name=Splunk OpenTelemetry Collector Repository
-baseurl=https://splunk.jfrog.io/splunk/otel-collector-rpm/release/\$basearch
-gpgcheck=1
-gpgkey=https://splunk.jfrog.io/splunk/otel-collector-rpm/splunk-B3CD4420.pub
-enabled=1
-EOH
-
-yum install -y splunk-otel-collector
-```
+  ```sh
+  $ curl -sSL https://splunk.jfrog.io/splunk/otel-collector-deb/splunk-B3CD4420.gpg > /etc/apt/trusted.gpg.d/splunk.gpg
+  $ echo 'deb https://splunk.jfrog.io/splunk/otel-collector-deb release main' > /etc/apt/sources.list.d/splunk-otel-collector.list
+  $ apt-get update
+  $ apt-get install -y splunk-otel-collector
```

Comment:
I'm not familiar with doc standards and am pretty inconsistent with this myself, but it can be nice to copy multiple commands for pasting at once and using `$` prevents this.

Reply:
I can remove the `$`.  Should we also chain these commands?

Reply:
I think removing the `$` is good enough but defer to @tmorrisonsfx

Reply:
Yes, you can remove the `$`. The copy function on the site will include the `$` by default.